src/controllers/discord/OAuth2Controller.ts

```diff
@@ -20,6 +21,12 @@ interface TokenResponse {
 	scope: string;
 }
 
+enum AuthRole {
```

Comment:
These are URIs for discord roles, right?

Reply:
Yep, I guess the enum could be named more clearly. Will update this tomorrow.

Reply:
Could you rename this to something like `DiscordRolesUri`. `AuthRole` implies that you're referring to the roles on `hs_auth`